pkg/assembler/graphql/schema/certifyVEXStatement.graphql

```diff
@@ -57,6 +57,8 @@ enum VexStatus {
   AFFECTED
   FIXED
   UNDER_INVESTIGATION
+  RESOLVED_WITH_PEDIGREE
```

Comment:
we decided not to add new ones but map to existing ones. Use `StatusNotes ` for storing the actual status stated by CDX.